src/main/java/org/terasology/logic/health/event/DoRestoreEvent.java

```diff
@@ -23,13 +24,20 @@
 public class DoRestoreEvent implements Event {
     /** The amount of health points being restored. */
     private int amount;
+    /** The instigator of restoration */
```

Comment:
Tiny suggestion: rather than use `instigator` which is already the variable name use a synonym to increase the odds that a reader will understand (instigator could be a somewhat rare word)

```suggestion
    /** The entity that caused the restoration */
```